gamearea.py

```diff
@@ -624,6 +624,10 @@ def get_next_level(self):
 
     def generate_score(self, reaction_time):
         score = self.score
+        if self.level == 1 and self.max_puzzle_count == 10 and self.win is False:
+            score = 0
+        if self.level == 1:
+            score = 20
```

Comment:
The score should only increase when a level is cleared and not because of a certain level.

Reply:
You will only get to know your score if you clear the level, it is acting as a storage thats it.
There are two ways to come out of game either you press star which will lead to new game or exhaust all the attempts which is take care of.

Reply:
And if you lose it will tell you to try again until you clear the level or exhaust all the attempts.

Reply:
Say a player fails at level one, that check automatically makes their score 20 when they failed.